src/normalizations/normalize-editor.ts

```diff
@@ -29,17 +29,29 @@ export const getVirtualRoom = (xprop: any): { label: string; link: string } | un
 	return undefined;
 };
 
-const getMeetingRooms = (attendees: Array<Attendee>): Array<{ email: string; label: string }> =>
-	map(filter(attendees, ['cutype', CALENDAR_RESOURCES.ROOM]), (at) => ({
-		label: at.d,
-		email: at.a
-	}));
-
-const getEquipments = (attendees: Array<Attendee>): Array<{ email: string; label: string }> =>
-	map(filter(attendees, ['cutype', CALENDAR_RESOURCES.RESOURCE]), (at) => ({
-		label: at.d,
-		email: at.a
-	}));
+const getMeetingRooms = (
+	attendees: Array<Attendee>
+): Array<{ email: string; label: string }> | undefined => {
+	const rooms = filter(attendees, ['cutype', CALENDAR_RESOURCES.ROOM]);
+	return rooms.length
+		? map(rooms, (at) => ({
+				label: at.d,
+				email: at.a
+		  }))
+		: undefined;
```

Comment:
to avoid confusion I would rather return null. undefined typically indicates that a value has not been assigned yet

Reply:
I disagree using null as if you use typeof null it will evaluate into an object.